src/StripeTests/StripeTests.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\netfx.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;netcoreapp2.1;netcoreapp3.0;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp3.1;net452</TargetFrameworks>
```

Comment:
why `netcoreapp2.0` suddenly when we had 2.1 before?

Reply:
I just want to make sure we're testing with the minimum version we're supporting. It shouldn't make any difference since .NET Core Runtime roughly adheres to semver, but better safe than sorry.